examples/oscilator_simulation/oscilator_simulation_example.m

```diff
@@ -88,7 +90,7 @@
 F = [f_11 f_12];
 model.F = F;
 %% Call integrator
-integrator = nosnoc.Integrator(model, problem_options, solver_options);
+integrator = nosnoc.Integrator(model, problem_options, integrator_options);
```

Comment:
this is the default call for an FESD integrator, but if I want a smoothed I need:
`integrator = nosnoc.integrator.SmoothedPss(model, problem_options, integrator_options);`

I would strongly prefer a unified syntax, i.e., an FESD integrator should be created via 
`integrator = nosnoc.integrator.FESD(model, problem_options, integrator_options);`

and so on.. otherwise it gets quickly too confusing...

Reply:
or maybe the better option is to have:
`integrator = nosnoc.Integrator(model, problem_options, integrator_options);`
and select the plugin via `integrator_options` . . .

Reply:
apparently, both are avilable and `nosnoc.Integrator` works fine, 
but `nosnoc.integrator.SmoothedPss` and `nosnoc.integrator.FESD` are broken. 

I prefer anyway to call it via `nosnoc.Integrator`   but the broken lower level should still be resolved.

Reply:
Yep, it should no longer be possible to instantiate the lower level plugins directly.